problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,30 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if mcq.value %} aria-label="{{mcq.value}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
```

Comment:
I wonder if the aria-label should give a little bit more information here, eg, "Score: mcq.value"